src/app/feed/info-box/info-box.component.ts

```diff
@@ -109,18 +105,29 @@ export class InfoBoxComponent implements OnInit, OnChanges {
 
 	}
 	sortMentions(statistics) {
-		let sortable = [];
+		let stored = [];
```

Comment:
What does the name `stored` signify here?

Reply:
```stored``` is the array storing the output sorted on basis of frequency.